js/level.js

```diff
@@ -96,29 +36,102 @@ function Level(specification) {
         container.style.left = -minX + "px";
         container.style.top = -minY + "px";
     };
-    this.getTileAt = function (x, y) {
+    this.getTileAt = function(x, y) {
         if (x < 0 || y < 0 || y >= this.tiles.length || x >= this.tiles[y].length) {
             return NullTile;
         }
         return this.tiles[y][x];
     };
-    this.setStartTile = function (startTile) {
+    this.setStartTile = function(startTile) {
         this.startTile = startTile;
     };
-    this.addPlayer = function (player) {
+    this.addPlayer = function(player) {
         player.startAt(this.startTile);
     };
-    this.visit = function () {
-        this.tiles.forEach(function (row) {
-            row.forEach(function (tile) {
+    this.visit = function() {
+        this.tiles.forEach(function(row) {
+            row.forEach(function(tile) {
                 tile.visit();
             });
         });
     };
     var me = this;
     this.startTile = null;
-    this.tiles = specification.map(function (row, y) {
-        return row.map(function (tileSpecification, x) {
+    this.tiles = specification.map(function(row, y) { // defined in /js/tiles.js
+        /* global NullTile */
+
+        const tileWidth = 429.544; // px, real width
```

Comment:
Is this a code duplication? This is not just a formatting.

Reply:
couldn't get you ?

Reply:
This part is a copy of the start. I will have a look at the file now.

Reply:
Yep: https://github.com/YashKumarVerma/labyrinth/blob/5decafb2ccc6880d54869f5147d693f085f5d2bf/js/level.js#L63

is the same as 
https://github.com/YashKumarVerma/labyrinth/blob/5decafb2ccc6880d54869f5147d693f085f5d2bf/js/level.js#L3

You copied parts over to other locations.

Reply:
This is not a formatting in my eyes.

Reply:
whops !

Reply:
What do you think?

Reply:
https://github.com/fossasia/labyrinth/pull/216

Reply:
i'm using github to do the changes, so sorry for opening another pr.